style.css

```diff
@@ -0,0 +1,71 @@
+.box__link {
+	cursor: pointer;
+	color: black;
+	padding: 15px 10px;
+	text-decoration: none;
+	text-align: center;
+	font-size: 36px;
+}
+
+.menu__mobile {
+	position: relative;
+	display: none;
+	left: 0;
+	top: 5px;
+	content: '';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Azez1/pullRequest?prid=5948917)